meta/labs/rejection-email.md

```diff
@@ -11,4 +13,5 @@ Leaders community: [https://www.facebook.com/groups/1501083703514499]
 Introduce yourself by sharing one challenge you know you'll face with your
 club this year.
 
-Looking forward to watching the coding community grow at your school, Dave
+Looking forward to watching the coding community grow at your school
+Dave
```

Comment:
Markdown renders this as a single line still. Would you please put a newline between line 16 and 17?